recipes/dropboxdrivefs/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dropboxdrivefs" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36eef7f7ad4d4b19f03d59d84d68326e950523882aec69674a1af8c14414c3a4"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
```

Comment:
Can this be a `noarch: python` package?
xref: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
you are right! I will add it. Thanks for the review.